### PR TITLE
Use pinned iree-tools-tflite in benchmarks

### PR DESCRIPTION
**Pull request by pzread**
Use pinned `iree-tools-tflite` to import tflite models for benchmarks and drop the build dependency on `build_tf_integrations`.

Doc to explain how to bump tf/tflite importer for benchmarks will be added in a follow-up change #13065

### CODE REVIEW

File: build_tools/cmake/setup_tf_python.sh [hunk 1]
```diff
@@ -14,4 +14,9 @@ export PATH="$PYTHON_SCRIPTS_DIR:$PATH"
...
 # Install local source-only Python packages. These do not have a build step
 # but export important binaries onto the path.
-"${IREE_PYTHON3_EXECUTABLE}" -m pip install integrations/tensorflow/python_projects/iree_tf
\ No newline at end of file
+"${IREE_PYTHON3_EXECUTABLE}" -m pip install integrations/tensorflow/python_projects/iree_tf
+
+# TODO(#13061): We should install from the local source-only Python packages
+# like iree-tools-tf, once we switch to Pyhton API based tflite importer.
+# Install pinned version of iree-tools-tflite.
+"${IREE_PYTHON3_EXECUTABLE}" -m pip install iree-tools-tflite=="20230412.487" -f https://openxla.github.io/iree/pip-release-links.html
```
Comment (stellaraccident) [inline build_tools/cmake/setup_tf_python.sh:22]: Add a newline at the end.

**Review by stellaraccident — APPROVED** [review]

File: build_tools/cmake/setup_tf_python.sh (hunk 1 above)
Comment (pzread) [reply build_tools/cmake/setup_tf_python.sh:22]: Oh, I think the diff is saying there was no a newline at the end, but now we have

**Comment by GMNGeoffrey** [comment]
Note: part of https://github.com/openxla/iree/issues/13037

I think it's going to cause some confusion that we have the source for `iree-import-tflite` in the repo but aren't using that, but severing this dependency is nice

**Comment by GMNGeoffrey** [comment]
Wait this also deletes `build_tf_integrations` entirely. `iree-import-tflite` is still being released so we do need a build of it running on CI

**Comment by pzread** [comment]
> Wait this also deletes `build_tf_integrations` entirely. `iree-import-tflite` is still being released so we do need a build of it running on CI

We still build `build_tf_integrations` in `ci.yml`. The one I removed is for long benchmarks workflow

**Comment by GMNGeoffrey** [comment]
> We still build `build_tf_integrations` in `ci.yml`. The one I removed is for long benchmarks workflow

Ah whew! Ok, thank you :-)

**Pull request merged**